api/resolvers/search.js

```diff
@@ -194,7 +194,16 @@ export default {
 
       const { query, quotes, nym, url } = queryParts(q)
 
-      if (url) {
+      if (url && !query.length) {
+        const domain = url.startsWith('url:www.') ? url.slice(8) : url.slice(4)
+        const value = `*${domain}*`
+        const fields = ['title', 'text']
+
+        const queries = fields
+          .map((field) => ({ wildcard: { [field]: { value, case_insensitive: true } } }))
+
+        whatArr.push({ bool: { should: queries } })
+      } else if (url) {
         whatArr.push({ match_phrase_prefix: { url: `${url.slice(4).toLowerCase()}` } })
```

Comment:
It might be worth doing a `wildcard` match here instead of `match_phrase_prefix`.

Reply:
Ended up removing this because the wildcard search on `url` in the filters removed all non-link results.